src/blog/2024/10/announcement-mqtt-broker.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "MQTT Service Now Available on FlowFuse"
+subtitle: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
+description: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
+date: 2024-10-30
+authors: ["joe-pavitt"]
+image: /blog/2024/10/images/ff-mqtt.png
+tags:
+   - posts
+   - news
+   - flowfuse
+   - node-red
+---
+
+You may have seen in our [recent product update](/blog/2024/10/flowfuse-release-2-10) that we have added our very own MQTT service, built-in and ready to use alongside your Node-RED applications. We are always engaging with users and prospective customers and this has been a highly requested feature, and so we are delighted to announce that this is now live on FlowFuse Cloud for our Team and Enterprise teams.
+
+<!--more-->
+
```

Comment:
MQTT is the lightweight messaging protocol ideal for efficient communication in industrial settings. Now with FlowFuse you can collect, transport, visualize and interact with your industrial data, all in one platform.